packages/wallet-lib/src/types/Storage/methods/configure.js

```diff
@@ -13,6 +16,18 @@ module.exports = async function configure(opts = {}) {
   this.autosave = has(opts, 'autosave') ? opts.autosave : this.autosave;
   this.adapter = await configureAdapter((opts.adapter) ? opts.adapter : await getDefaultAdapter());
 
+  const version = await this.adapter.getItem('version');
+
+  if (!version) {
+    await this.adapter.setItem('version', CURRENT_VERSION);
+  } else if (version !== CURRENT_VERSION) {
+    logger.warn('Storage validation error: schema mismatch: unknown version');
```

Comment:
Please change to `logger.warn('Storage version mismatch, resyncing from start')`